src/output/tecplotIO.F90

```diff
@@ -586,22 +600,28 @@ subroutine writeLiftDistributions(sps, fileID, nodalValues)
        end if
 
        allocate(values(d%nSegments, nLiftDistVar))
+       !MHAM
+       ! this 'values'-container seems to be very large. It must be holding
+       ! nSegments: Number of nodes to use for distribution (l. 31 above)
+       ! integer(kind=intType), parameter :: nLiftDistVar=21 (l. 64 above)
        values = zero
 
-       do i=1,d%nSegments
-          if (i==1) then
+       do i=1,d%nSegments !mham: do for each point in the distribution
```

Comment:
you can remove the extra comments added for this loop and logicals inside. useful, but not super necessary